src/plugins/intel_cpu/src/onednn/iml_type_mapper.h

```diff
@@ -111,6 +112,7 @@ enum impl_desc_type : int64_t {
 
     dw_acl = _dw | acl,
     gemm_acl = gemm | acl,
+    lowp_gemm_acl = lowp_gemm | acl,
```

Comment:
I don't really see big value in impl types extension in that case. Since we don't have any other options for int8 gemm, we can keep using gemm_acl type for it.

Reply:
Removed